src/codeu/chat/ClientMain.java

```diff
@@ -52,6 +53,31 @@ public static void main(String [] args) {
 
     LOG.info("Created client");
 
+    /// SQLite
+
+    Connection c = null; /// Create connection to database
+    Statement stmt = null; /// Placeholder for sql command
+    try {
+      Class.forName("org.sqlite.JDBC");
+      c = DriverManager.getConnection("jdbc:sqlite:test.db"); /// Connect to database. Creates database if it doesn't exist. 
+      System.out.println("Opened database successfully");
+
+      stmt = c.createStatement();
+      /// Command to create USER table
```

Comment:
Should we also create a Message table (and possibly a conversation table)? This looks like this would only allow users to be persistent but without Message SQL table the messages will not be persistent.